admin/component/errors/ErrorDisplay.tsx

```
@@ -0,0 +1,63 @@
+        <div className="inline-flex h-24 w-24 items-center justify-center rounded-full bg-red-100 mb-6">
+          <AlertCircle className="h-12 w-12 text-red-600" />
+        </div>
+        <h2 className="text-2xl font-semibold text-gray-900 mb-2">
+          {errorContent.title}
+        </h2>
+        <p className="text-gray-600 mb-6">{errorContent.message}</p>
+        {showBackButton && (
+          <button
+            onClick={() => router.back()}
+            className="inline-flex items-center px-4 py-2 border border-gray-300 shadow-sm text-sm font-medium rounded-md text-gray-700 bg-white hover:bg-gray-50"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)